packages/suite/src/components/suite/AppNavigation/index.tsx

```diff
@@ -98,6 +100,33 @@ const Text = styled.div`
     position: relative;
 `;
 
+const TextHover = styled.div`
+    position: relative;
+    &:after {
+        content: '';
+        position: absolute;
+        top: -4px;
+        left: -4px;
+        right: -4px;
+        bottom: -4px;
+        border-radius: 8px;
```

Comment:
According to the new designs, the border-radius also seems to start to be more unified, I'd also put it in the theme/config variables and later reuse it in other components as well.

Reply:
done

Reply:
dtto https://github.com/trezor/trezor-suite/pull/3829#discussion_r655227034